packages/api-sdk/src/shared.ts

```diff
@@ -16,9 +17,12 @@ export const optionalString = (zodSchema: ZodString) =>
 export const ISO_DATE = "YYYY-MM-DD";
 export const defaultString = z.string().trim();
 export const defaultOptionalString = optionalString(defaultString);
-export const defaultDateString = defaultString.refine(v => dayjs(v, ISO_DATE, true).isValid(), {
+export const defaultDateString = defaultString.refine(isValidISODate, {
   message: `Date must be a valid ISO 8601 date formatted ${ISO_DATE}. Example: 2023-03-15`,
 });
+export const pastOrTodayDateString = defaultDateString.refine(validateIsPastSafe, {
+  message: `Date can't be in the future`,
```

Comment:
`Date must be in the past`?

Reply:
It can be on the current date, so I thought this is clearer. Suggestions?

Reply:
It's clear, just wanted to align the message with the error that `validateIsPast` throws (just to have consistency) -- left comment on that section as then that error message should match this.